requirements.txt

```diff
@@ -3,6 +3,7 @@ shapely>=1.2.18
 fiona>=1.0.1
 pyproj>=1.9.3
 six>=1.3.0
+pandas>=0.15.2
```

Comment:
this is not needed. 
This requirements file is used on travis, and the pandas version is specified in another way (adding it here will cause it to be installed twice, making the travis build to take longer)